add example that plays two chords on PSG

pinobatch opened the pull request:
Demonstrate use of the SN7 PSG by playing two chords (E flat major and C major) that should be familiar to most Mega Drive owners.

pinobatch committed:
add example that plays two chords on PSG

Demonstrate use of the SN7 PSG by playing two chords (E flat major
and C major) that should be familiar to most Mega Drive owners.

Mikejmoffitt commented on psg/src/main.c:
```diff
@@ -0,0 +1,78 @@
+	     ++playpos)
+	{
+		// Set pitch
+		for (unsigned int ch = 0; ch < 3; ++ch) {
```
Within a single project (e.g. the scope of one sample) I'm okay with the code style diverging from my own, since this example is yours, and not mine. Here, my only complaint is that it'd be best for it to be internally consistent.

In this case, the opening brace is on the same line as the `for` statement, while the loop below has it on the next line instead.

Between the two, I'd prefer if you picked your favorite and used it for the file. I lean towards the latter, and have thus far in the framework code.

pinobatch replied:
Oops, my bad. I tried to follow your style but missed these.

Mikejmoffitt commented on psg/src/main.c:
```diff
@@ -0,0 +1,78 @@
+		}
+	}
+
+	for (unsigned int ch = 0; ch < 3; ++ch) {
```
Repeat of the comment from L54.

Mikejmoffitt commented on psg/src/main.c:
```diff
@@ -0,0 +1,78 @@
+// md-toolchain example main.c
+// Michael Moffitt 2018
+//
+// This main shows a simple "hello world" demo.
```
This comment block is a leftover from when this started its life as hello world.

pinobatch replied:
PR updated

pinobatch committed:
address indentation issue

PR #7 pointed out 2 lines whose indent style was inconsistent with
that of the framework's .c files, which appears to be as follows:

- indent 1 tab
- left brace at same level as preceding if/while/for
- else on its own line
